src/views/SafeAreaView.js

```diff
@@ -76,11 +76,7 @@ const statusBarHeight = isLandscape => {
     return isLandscape ? 0 : 44;
   }
 
-  if (isIPad) {
```

Comment:
Why did you delete this? I think iPad landscape mode still has 20px status bar

Reply:
Because its unnecessary part of code... now we always return 20px..

Reply:
And the test is unnecessary too... all non-iPhone X devices have status bar, only iPhone X in landscape mode doesn't... so it works

Reply:
Oops, I misread that. So, its always 20px? What about smaller phone sizes or older versions of iOS? If you think this is safe then lets merge!

Reply:
Ehm, i just found it was some bug on my simulator... because the status bar should hide on landscape mode.. so your code is correct, i apologize